JSDemos/Demos/DataGrid/EditStateManagement/Vue/App.vue

```diff
@@ -60,6 +60,7 @@ import { State } from './store.ts';
 
 const loadPanelPosition = { of: '#gridContainer' };
 
+// it's really need
```

Comment:
We can't show this to user :)

Reply:
Also, why we disabled react lint rule for one line in vue code instead of disabling react rules in vue code in general?